src/esp/physics/RigidObject.h

```diff
@@ -507,6 +507,11 @@ class RigidObject : public Magnum::SceneGraph::AbstractFeature3D {
   //! drawing is off. See @ref toggleBBDraw().
   scene::SceneNode* BBNode_ = nullptr;
 
+  /**
+   * @brief All Drawable components are children of this node.
+   */
+  scene::SceneNode* visualNode_ = nullptr;
```

Comment:
any reason this is public versus having a getter (like `node()`)?

Reply:
Reasoning: Currently,  `PhysicsManager` needs access to these pointers and they can be null, so references aren't ideal. The getters would then be for raw pointer anyway. Users should not be getting direct access to a RigidObject to manipulate these, so I left them at public.